File: blog-post-2/cpp/PrimeNumber.cpp
#include <iostream>
using namespace std;
int main()
{
    int number,isPrime,i;
    cout <<"Enter number";
    cin >> number;
    isPrime = 1;
    if (number != 2)
    {
        for (int i = 2; i <= number / 2; i++)
        {
            if (number % i == 0)
            {
                isPrime = 0;
                break;
            }
        }
    }

    if (isPrime)
    {
        cout <<"%d is prime number",number;
    }
    else
    {
        cout <<"%d is not prime number",number;
    }
    return 0;
}

File: blog-post-2/programjava/PrimeNumber.java
package programjava;

import java.util.Scanner;

public class PrimeNumber {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.print("Enter number");
        int number = in.nextInt();
        boolean primeNumber = true;
        in.close();
        if (number != 2) {
            for (int i = 2; i <= number / 2; i++) {
                if (number % i == 0) {
                    primeNumber = false;
                    break;
                }
            }
        }
        
        if (primeNumber) {
            System.out.println(number + " is prime number");
        } else {
            System.out.println(number + " is not prime number");
        }
    }
}